capture-sdk/sdk/src/main/java/net/gini/android/capture/internal/camera/photo/PhotoEdit.java

```diff
@@ -73,6 +85,39 @@ private void removeCompressionModifier() {
         }
     }
 
+    public PhotoEdit crop(Activity context, Rect aRect) {
+
+        byte[] originalBytes = mPhoto.getData();
+
+        try {
+            Bitmap originalBitmap = BitmapFactory.decodeByteArray(originalBytes,
+                    0, mPhoto.getData().length);
+
+            DisplayMetrics metrics = new DisplayMetrics();
+            context.getWindowManager().getDefaultDisplay().getMetrics(metrics);
+
+            int x1 = originalBitmap.getWidth() * aRect.left / metrics.widthPixels;
```

Comment:
[Request] The bitmap can be rotated and we need to account for that. The `mPhoto.getRotationForDisplay()` can be used to find out how the bitmap needs to be rotated to be in the right orientation for the cropping to work.

Rotation is always by 90 degree steps so this switch will cover all cases:
```
            switch (mPhoto.getRotationForDisplay() % 360) {
                case 0:
                    // bitmap and aRect have same rotation
                    break;
                case 90:
                    // bitmap is rotated 90 CCW compared to aRect
                    break;
                case 180:
                    // bitmap is upside-down compared to aRect
                    break;
                case 270:
                    // bitmap is rotated 270 CCW compared to aRect
                    break;
                default:
                    // this should never happen
                    break;
            }
```

For each case the crop values x1, y1, width1 and height1 need to account for the rotation.

If we don't then for example on the Galaxy S21 the cropping is wrong because the `getRotationForDisplay()` is `90` degrees:
![Screenshot_20221021_182651](https://user-images.githubusercontent.com/13149498/197243974-5cc45280-af72-4b9c-9a9a-70640074ca75.png)
![Screenshot_20221021_182710](https://user-images.githubusercontent.com/13149498/197243998-ff5f919e-e6e1-4f9e-bca5-fee20c1c7111.png)